create_provenance.go

```diff
@@ -222,22 +221,9 @@ func main() {
 		stmt.Predicate.Builder.Id = repoURI + SelfHostedIdSuffix
 	}
 
-	stmtPayload, _ := json.MarshalIndent(stmt, "", "  ")
-	fmt.Println("Payload:\n" + string(stmtPayload))
-	if err := ioutil.WriteFile(*outputPath+".payload", stmtPayload, 0755); err != nil {
-		fmt.Println("Failed to write provenance payload: %s", err)
-		os.Exit(1)
-	}
-
-	stmtCompactPayload, _ := json.Marshal(stmt)
-	envelope := Envelope{
-		PayloadType: PayloadContentType,
-		Payload:     base64.StdEncoding.EncodeToString(stmtCompactPayload),
-		Signatures:  []interface{}{},
-	}
-	envelopePayload, _ := json.MarshalIndent(envelope, "", "  ")
-	fmt.Println("Provenance:\n" + string(envelopePayload))
-	if err := ioutil.WriteFile(*outputPath, envelopePayload, 0755); err != nil {
+	payload, _ := json.MarshalIndent(stmt, "", "  ")
```

Comment:
Is it worth a comment that says why we're just writing the statement and that at L2+ it will need to be signed?

Reply:
Done.